pkg/drivers/kic/oci/oci.go

```diff
@@ -84,6 +84,18 @@ func DeleteContainer(ociBin string, name string) error {
 	return nil
 }
 
+// SetupContainerNode sets up the container node
+func SetupContainerNode(p CreateParams) error {
```

Comment:
this doesnt seem to be creating a container, did you mean crate Volume?

Reply:
so creating this volume currently happens in `CreateContainerNode`, but I wanted it to happen ASAP so I moved it into this function `SetupContainerNode` to make it clear that it needs to run before `CreateContainerNode`

Reply:
how about adding a comment to explain what this does, so future contributors know what and why.

and also maybe a different name, how about
prepareContainer
prepareNodeVolume

Reply:
done